### ISSUE TITLE
Consider cogs for pushing up the full reports

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/OBO-Dashboard#40]
In addition to the truncated reports, we could push the full robot reports up with cogs to a google docs.. It would be quite some google docs, with about 150 sheets, but may be worth a try.

**Comment by jamesaoverton** [comment]
I don't think this will work for the larger reports. COGS uses the Google Sheets API, which is designed for working on cells and ranges. I've found that slow to push even 10MB of data.

I've also used [rclone](https://rclone.org/) to push and pull files to Google Sheets / Drive. It can also push to most cloud providers.

**Comment by matentzn** [comment]
Ah I didnt know of that limitation! Thanks for enlightening me!
